vertical-pod-autoscaler/pkg/admission-controller/resource/vpa/handler.go

```diff
@@ -133,9 +136,17 @@ func validateVPA(vpa *vpa_types.VerticalPodAutoscaler, isCreate bool) error {
 				}
 			}
 			for resource, min := range policy.MinAllowed {
+				if err := validate(resource, min); err != nil {
+					return err
```

Comment:
It would be nice to mention in the error message which field has improper value.

You can add this information here:

```
  return fmt.Errorf("MinAllowed: %s", err)
```

or pass the name of the field to `validate` so it can include it in the error message it returns.